dev/RadioMenuFlyoutItem/RadioMenuFlyoutItem.vcxitems

```diff
@@ -21,54 +21,18 @@
     <ClCompile Include="$(MSBuildThisFileDirectory)RadioMenuFlyoutItem.cpp" />
   </ItemGroup>
   <ItemGroup >
-    <Page Include="$(MSBuildThisFileDirectory)RadioMenuFlyoutItem_rs1_themeresources.xaml">
+    <Page Include="$(MSBuildThisFileDirectory)RadioMenuFlyoutItem_themeresources.xaml">
       <ControlsResourcesVersion>Version2</ControlsResourcesVersion>
       <Version>RS1</Version>
       <Type>ThemeResources</Type>
       <IsPublic>false</IsPublic>
     </Page>
-    <Page Include="$(MSBuildThisFileDirectory)RadioMenuFlyoutItem_rs1_themeresources_v1.xaml">
+    <Page Include="$(MSBuildThisFileDirectory)RadioMenuFlyoutItem_themeresources_v1.xaml">
       <ControlsResourcesVersion>Version1</ControlsResourcesVersion>
       <Version>RS1</Version>
       <Type>ThemeResources</Type>
       <IsPublic>false</IsPublic>
     </Page>
-    <Page Include="$(MSBuildThisFileDirectory)RadioMenuFlyoutItem_rs2_themeresources.xaml">
-      <ControlsResourcesVersion>Version2</ControlsResourcesVersion>
-      <Version>RS2</Version>
-      <Type>ThemeResources</Type>
-      <IsPublic>false</IsPublic>
-    </Page>
-    <Page Include="$(MSBuildThisFileDirectory)RadioMenuFlyoutItem_rs2_themeresources_v1.xaml">
-      <ControlsResourcesVersion>Version1</ControlsResourcesVersion>
```

Comment:
you should leave the version 1 files alone and just merge the version 2 ones. we dont want to touch the old styles.